pingrid.py

```diff
@@ -540,9 +540,13 @@ def average_over(ds, s, lon_name="lon", lat_name="lat", all_touched=False):
     r = r0 * np.cos(np.deg2rad(ds[lat_name]))
 
     norm = r / r.sum([lat_name, lon_name])
-    res = (ds * norm).sum([lat_name, lon_name], skipna=True)
+    res = (ds * norm).sum([lat_name, lon_name], skipna=True, min_count=1)
```

Comment:
what is the significance of this `min_count`?

Reply:
This fixed the bug where years with no NDVI score were being counted as having an NDVI of zero instead of missing. If da is all NaNs, `da.sum(skipna=True)` returns zero whereas `da.sum(skipna=True, min_count=1)` returns NaN unless there's at least one non-missing value.